sql/healthfacilities.sql

```diff
@@ -0,0 +1,1650 @@
+10,district,HEALTH_CENTER,0
+10,district,CENTRAL_HOSPITAL,0
+10,district,HEALTH_CARE_CENTER,0
+10,district,HOSPITAL,1
+10,district,SUB_CENTER,1
+10,district,LAXMIPUR,0
+10,district,AYURVEDIC_AUSHADHALAYA,0
+10,district,HEALTH_POST,7
+10,district,PRIVATE_HOSPITAL,0
+10,district,DISTRICT_COLD_ROOM,1
+10,district,ZONAL_HOSPITAL,0
+10,district,DISTRICT_CENTER,0
+10,district,SUB_HEALTH_POST,48
+10,district,REFUGEE_CAMP,0
+10,district,DISTRICT_AYURVEDIC_HC,0
+10,district,DAHC,0
+10,district,DPHO,1
+39,district,RMS,0
+39,district,SUPPLY_CENTER,4
+39,district,PRIMARY_HEALTH_POST,0
+39,district,PRIMARY_HEALTH_CENTER,0
+39,district,HEALTH_CENTER,0
+39,district,CENTRAL_HOSPITAL,0
+39,district,HEALTH_CARE_CENTER,0
+39,district,HOSPITAL,1
+39,district,SUB_CENTER,0
+39,district,LAXMIPUR,0
+39,district,AYURVEDIC_AUSHADHALAYA,0
+39,district,HEALTH_POST,8
+39,district,PRIVATE_HOSPITAL,0
+39,district,DISTRICT_COLD_ROOM,1
+39,district,ZONAL_HOSPITAL,0
+39,district,DISTRICT_CENTER,0
+39,district,SUB_HEALTH_POST,4
+39,district,REFUGEE_CAMP,0
+39,district,DISTRICT_AYURVEDIC_HC,0
+39,district,DAHC,0
+39,district,DPHO,1
+38,district,RMS,0
+38,district,SUPPLY_CENTER,0
+38,district,PRIMARY_HEALTH_POST,0
+38,district,PRIMARY_HEALTH_CENTER,1
+38,district,HEALTH_CENTER,0
+38,district,CENTRAL_HOSPITAL,0
+38,district,HEALTH_CARE_CENTER,0
+38,district,HOSPITAL,2
+38,district,SUB_CENTER,3
+38,district,LAXMIPUR,0
+38,district,AYURVEDIC_AUSHADHALAYA,0
+38,district,HEALTH_POST,13
+38,district,PRIVATE_HOSPITAL,0
+38,district,DISTRICT_COLD_ROOM,1
+38,district,ZONAL_HOSPITAL,0
+38,district,DISTRICT_CENTER,0
+38,district,SUB_HEALTH_POST,31
+38,district,REFUGEE_CAMP,0
+38,district,DISTRICT_AYURVEDIC_HC,0
+38,district,DAHC,0
+38,district,DPHO,1
+15,district,RMS,0
+15,district,SUPPLY_CENTER,12
+15,district,PRIMARY_HEALTH_POST,0
+15,district,PRIMARY_HEALTH_CENTER,4
+15,district,HEALTH_CENTER,0
+15,district,CENTRAL_HOSPITAL,0
+15,district,HEALTH_CARE_CENTER,0
+15,district,HOSPITAL,0
+15,district,SUB_CENTER,0
+15,district,LAXMIPUR,0
+15,district,AYURVEDIC_AUSHADHALAYA,0
+15,district,HEALTH_POST,9
+15,district,PRIVATE_HOSPITAL,0
+15,district,DISTRICT_COLD_ROOM,1
+15,district,ZONAL_HOSPITAL,1
+15,district,DISTRICT_CENTER,0
+15,district,SUB_HEALTH_POST,117
+15,district,REFUGEE_CAMP,0
+15,district,DISTRICT_AYURVEDIC_HC,0
+15,district,DAHC,0
+15,district,DPHO,1
+14,district,RMS,0
+14,district,SUPPLY_CENTER,76
+14,district,PRIMARY_HEALTH_POST,0
+14,district,PRIMARY_HEALTH_CENTER,152
+14,district,HEALTH_CENTER,0
+14,district,CENTRAL_HOSPITAL,0
+14,district,HEALTH_CARE_CENTER,0
+14,district,HOSPITAL,76
+14,district,SUB_CENTER,228
+14,district,LAXMIPUR,0
+14,district,AYURVEDIC_AUSHADHALAYA,0
+14,district,HEALTH_POST,684
+14,district,PRIVATE_HOSPITAL,0
+14,district,DISTRICT_COLD_ROOM,76
+14,district,ZONAL_HOSPITAL,0
+14,district,DISTRICT_CENTER,0
+14,district,SUB_HEALTH_POST,2736
+14,district,REFUGEE_CAMP,0
+14,district,DISTRICT_AYURVEDIC_HC,0
+14,district,DAHC,0
+14,district,DPHO,76

```

Comment:
The numbers for district 14 seem off. There are 684 health posts and 2736 sub health posts listed for district 14. That would be about half of the sub health posts in the country. Can you verify these figures?

Reply:
Yes, good spot....I'll check.

Reply:
Yes, there are 2736 unique entries in the shapefile for 'Sub Health Post' and 'Udayapur' however it seems like those entries are just these 36 repeated 76 times:

```
[0, 'Sub Health Post', 'Udayapur', 'Katari', 'E-SAG-14-017'],
[0, 'Sub Health Post', 'Udayapur', 'Risku', 'E-SAG-14-030'],
[0, 'Sub Health Post', 'Udayapur', 'Tawashree', 'E-SAG-14-039'],
[0, 'Sub Health Post', 'Udayapur', 'Okhale', 'E-SAG-14-026'],
[0, 'Sub Health Post', 'Udayapur', 'Limpatar', 'E-SAG-14-023'],
[0, 'Sub Health Post', 'Udayapur', 'Sirise', 'E-SAG-14-035'],
[0, 'Sub Health Post', 'Udayapur', 'ShorungChabise', 'E-SAG-14-033'],
[0, 'Sub Health Post', 'Udayapur', 'Lekhani', 'E-SAG-14-021'],
[0, 'Sub Health Post', 'Udayapur', 'Thanagau', 'E-SAG-14-040'],
[0, 'Sub Health Post', 'Udayapur', 'Basabote', 'E-SAG-14-005'],
[0, 'Sub Health Post', 'Udayapur', 'Tamlida', 'E-SAG-14-037'],
[0, 'Sub Health Post', 'Udayapur', 'Baraha', 'E-SAG-14-003'],
[0, 'Sub Health Post', 'Udayapur', 'Iname', 'E-SAG-14-013'],
[0, 'Sub Health Post', 'Udayapur', 'Barai', 'E-SAG-14-004'],
[0, 'Sub Health Post', 'Udayapur', 'Nametar', 'E-SAG-14-025'],
[0, 'Sub Health Post', 'Udayapur', 'Dumre', 'E-SAG-14-010'],
[0, 'Sub Health Post', 'Udayapur', 'Jante', 'E-SAG-14-015'],
[0, 'Sub Health Post', 'Udayapur', 'Laphagau', 'E-SAG-14-020'],
[0, 'Sub Health Post', 'Udayapur', 'Pokhari', 'E-SAG-14-028'],
[0, 'Sub Health Post', 'Udayapur', 'Valayadanda', 'E-SAG-14-044'],
[0, 'Sub Health Post', 'Udayapur', 'Aaptar', 'E-SAG-14-001'],
[0, 'Sub Health Post', 'Udayapur', 'Khanbu', 'E-SAG-14-019'],
[0, 'Sub Health Post', 'Udayapur', 'Jalpachilaune', 'E-SAG-14-014'],
[0, 'Sub Health Post', 'Udayapur', 'Saune', 'E-SAG-14-032'],
[0, 'Sub Health Post', 'Udayapur', 'Sidhdipur', 'E-SAG-14-034'],
[0, 'Sub Health Post', 'Udayapur', 'Hadiya', 'E-SAG-14-011'],
[0, 'Sub Health Post', 'Udayapur', 'Jogidaha', 'E-SAG-14-016'],
[0, 'Sub Health Post', 'Udayapur', 'Sundarpur', 'E-SAG-14-036'],
[0, 'Sub Health Post', 'Udayapur', 'Bashaha', 'E-SAG-14-006'],
[0, 'Sub Health Post', 'Udayapur', 'Chaudandi', 'E-SAG-14-009'],
[0, 'Sub Health Post', 'Udayapur', 'Katunjebawala', 'E-SAG-14-018'],
[0, 'Sub Health Post', 'Udayapur', 'Tapeswori', 'E-SAG-14-038'],
[0, 'Sub Health Post', 'Udayapur', 'Koshi Tappu Wildlife', 'E-SAG-14-046'],
[0, 'Sub Health Post', 'Udayapur', 'Mainamiani', 'E-SAG-14-024'],
[0, 'Sub Health Post', 'Udayapur', 'Hardeni', 'E-SAG-14-012'],
[0, 'Sub Health Post', 'Udayapur', 'Tribeni', 'E-SAG-14-042'],
[0, 'Sub Health Post', 'Udayapur', 'Katari', 'E-SAG-14-017'],
[0, 'Sub Health Post', 'Udayapur', 'Risku', 'E-SAG-14-030'],
```

I'm not sure whether we should just include these 36?

Reply:
Looking at the map, it does seem like 36 should be the figure; I'll check whether any of the other regions have duplicates like this.

Reply:
Removing what appear to be duplicates seems reasonable. 

Several of the other facility types look like they may have repeats in this district, too: hospital, sub center, health post, district cold room, DPHO all have large numbers.

Can you change how the script processes data so it enforces uniqueness?